Projekty/Projekt1/AdriannaGrudzień_KingaUłasik/WUM_projekt1_inżynieria_cech.ipynb

```diff
@@ -0,0 +1,1810 @@
+{
```

Comment:
<p>Nie powiedziałem Wam tego, ale też nie pytałyście, więc podzielmy się&nbsp;winą po połowie \:)</p><p> </p><p>https://www.researchgate.net/publication/228780408_Using_data_mining_to_predict_secondary_school_student_performance</p><p> </p><p>To jest paper, z którego wywodzą się te dane. Oni tam robili klasyfikację binarną i 5 klasową oraz regresję. To może być inspiracja</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Przejrzałem inne prace i niektórzy usuwają rekordy z G3 = 0, możecie się nad tym zastanowić. Może ma to sens, a może nie</p>

Reply:
<p>Jak poprawicie random_state to dam Wam 4</p>